src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -178,6 +178,26 @@ public function __construct($strTable)
 			$this->arrValidFileTypes = \StringUtil::trimsplit(',', strtolower($GLOBALS['TL_DCA'][$this->strTable]['config']['validFileTypes']));
 		}
 
+		// Configure the picker
```

Comment:
I have checked it and it does not really make sense to move any code into the parent class. Especially as we would have to add another abstract method `setPickerValue()`, which will break compatibility with all existing DCs.